### PR TITLE
[4.1] CANDLEPIN-740: Updated RH packaged certificate

### PR DESCRIPTION
**Pull request by joshmalbrecht**
- updated the certificate to be the same as the one in main

I was able to verify the changes by creating an export from the stage environment and successfully importing the manifest using the `/owners/<owner-key>/imports/async` endpoint

### CODE REVIEW

File: config/candlepin/candlepin-redhat-ca.crt [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+jKdh2oJhvfRnl6h92hahxjdf2dPPBxIT/Rf2IUB8/axFOKP1hPnLz7NgmITB/cKe
+LwrskG+DCaWFVEAwCW3PbvQyvcfW2AZQOx6vQZIwmR3FmJBX/A3XNF/4CciStcIH
+irhtmiH4WY3TiOtX0V8Jy1z10SHFm3NZeK4S1lqf3fPmsgMwecqBK+bVIvOavCSD
+tNOlIdvB69FxBv0uTxbW3jxxYJXQyENeNpi9mcSsAg725s+hi99DolTJ4qvaraOA
+9ECbeR7zf++oTMDXm20I8wyskvHENCV8z/aQmZ1ukNejXoj0X6Li0hZraqL8nZ31
+XbQlrEBew5ikJcaqab7/H+Hl2w1oNZENh/31sw9t/NZGJd9N7zS9kVtgr16b138P
+7EXJFHWHFZvQD3iuFbN38EgWzDAY0DPpiMQZ7sa0D+hl0j/T5tauGGQ9qKT70FtL
+ym8oHWwytyfTU2cF1ivzig3DSKOGOLDZr2o7zh/Q4eCzPYfk4ieWfYsd4rRB6+Y4
+E6/lvbR33zc=
+-----END CERTIFICATE-----
```
Comment (Ceiu) [inline config/candlepin/candlepin-redhat-ca.crt:40]: The trailing line break got lost in the update; we'll want to restore that for CLI output friendliness.

**Review by Ceiu — CHANGES_REQUESTED** [review]
Few minor changes and one important one:
- the missing new line at the end of the new cert
- missing line break in the commit message between the title and the body
- gencerts needs to be updated to no longer reference the r2-ca.crt file

**Review by nikosmoum — COMMENTED** [review]
based on testing, this change shouldn't be needed. lets wait for a clear reproduction of the issue, to find the root cause

**Comment by joshmalbrecht** [comment]
Closing, as the issue appears to be that the customer's upstream dir was moved and the certificates could not be found.

**Pull request closed without merging**